packages/goto/src/index.js

```diff
@@ -14,17 +15,30 @@ const isEmpty = val => val == null || !(Object.keys(val) || val).length
 
 const WAIT_UNTIL = ['networkidle0']
 
+const parseCookies = (url, str) => {
+  const domain = `.${getDomain(url)}`
```

Comment:
@Kikobeats When it comes to cookie matters, I think you might want to also take into account "[private suffixes](https://github.com/publicsuffix/list/blob/master/public_suffix_list.dat#L10669)" like so:
```js
getDomain(url, { allowPrivateDomains: true })
```